final_prep/README.md

```diff
@@ -32,9 +32,9 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![Week 1](https://user-images.githubusercontent.com/79383963/114285092-aa775180-9a11-11eb-8804-768766f583b4.png)
```

Comment:
Your calendars look really good! I see it looks like your hard cutoff every day is 9:15 - make sure you communicate that with project partners. Your YOU time is super important!